pkgs/development/interpreters/kamilalisp/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, jre, lib, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "kamilalisp";
+  version = "0.2";
+
+  src = fetchurl {
+    url = "https://github.com/kspalaiologos/kamilalisp/releases/download/v0.2o/kamilalisp-0.2.jar";
+    sha256 = "sha256-NkRkLcJpar9iHEI+OmgpkYwkwVvU1J7izeBdytwIsdU=";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+  mkdir -pv $out/share/java $out/bin
+  cp ${src} $out/share/java/${name}-${version}.jar
+  makeWrapper ${jre}/bin/java $out/bin/kamilalisp \
+    --add-flags "-jar $out/share/java/${name}-${version}.jar" \
+    --set _JAVA_OPTIONS '-Dawt.useSystemAAFontSettings=on' \
+    --set _JAVA_AWT_WM_NONREPARENTING 1
+  '';
+
+  meta = {
+    homepage = "https://github.com/kspalaiologos/kamilalisp.git";
+    description = "A functional, flexible, and concise Lisp.";
+    license = lib.licenses.gpl3Plus;
+    platforms = lib.platforms.all;
+    maintainers = with lib.maintainers;
+      [ cafkafk ];
+  };
```

Comment:
```suggestion
    sourceProvenance = [ lib.sourceTypes.binaryBytecode ];
  };
```

Reply:
ohh didn't know about this, that's neat!